### PR TITLE
Include columns in lint range

### PR DESCRIPTION
**Pull request by b52**
Fixes issue #21

### CODE REVIEW

File: lib/linter-rust.coffee [hunk 1]
```diff
@@ -36,8 +36,8 @@ class LinterRust
             text: match.message
             filePath: if path.isAbsolute match.file then match.file else path.join curDir, match.file
             range: [
-              [match.line - 1, 0],
-              [match.line - 1, 0]
+              [match.from_line - 1, match.from_col - 1],
+              [match.to_line - 1, match.to_col - 1]
             ]
```
Comment (steelbrain) [inline lib/linter-rust.coffee]: you don't need a `-1` on the end col. cause in case the start and end col is the same, it won't highlight anything.

**Comment by b52** [comment]
Fixed your remarks @steelbrain

**Comment by steelbrain** [comment]
LGTM
/cc @gmist

File: lib/linter-rust.coffee [hunk 2]
```diff
@@ -25,19 +25,23 @@ class LinterRust
...
         messages = []
-        regex = XRegExp('(?<file>.+):(?<line>\\d+):(?<col>\\d+):\\s*(\\d+):(\\d+)\\s+((?<error>error|fatal error)|(?<warning>warning)|(?<info>note)):\\s+(?<message>.+)\n', '')
+        regex = XRegExp('(?<file>.+):(?<from_line>\\d+):(?<from_col>\\d+):\\s*(?<to_line>\\d+):(?<to_col>\\d+)\\s+((?<error>error|fatal error)|(?<warning>warning)|(?<info>note)):\\s+(?<message>.+)\n', '')
         XRegExp.forEach results.join(''), regex, (match) =>
           type = if match.error
             "Error"
           else if match.warning
             "Warning"
+
+          if match.from_col == match.to_col
```
Comment (steelbrain) [inline lib/linter-rust.coffee]: SideNote: `===` should be preferred over `==`

Comment (b52) [inline lib/linter-rust.coffee]: In JS, yes, but not in CS, [it does that for you](http://coffeescript.org/#try:foo%20%3D%20123%0Abar%20%3D%20%27123%27%0A%0Aif%20foo%20%3D%3D%20bar%0A%20%20alert%20%27gnah%27).

Comment (steelbrain) [inline lib/linter-rust.coffee]: That's new, thanks!

**Pull request merged**